application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -146,6 +132,40 @@ private boolean isUserOnCooldown(User user) {
             .isPresent();
     }
 
+    private void sendCooldownResponse(SlashCommandInteractionEvent event) {
+        User user = event.getUser();
+
+        String message =
+                """
+                        Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread %s instead.
+                        If you made a typo or similar, you can adjust the title using the command %s and the category with %s :ok_hand:""";
+
+        RestAction<String> changeTitle = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
+                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
+                HelpThreadCommand.CHANGE_TITLE_SUBCOMMAND);
+        RestAction<String> changeCategory = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
+                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
+                HelpThreadCommand.CHANGE_CATEGORY_OPTION);
```

Comment:
duplication. might want to create a helper method `mentionHelpChangeCommand(String subcommand)` right here in this class.